rdb_coord speed
rdb_coord accel
float32 traveledDist
uint32[3] spare